R/registry.R

```diff
@@ -62,15 +62,19 @@ get_status <- function(entry) {
 }
 
 guess_status <- function(entry) {
-  if ("codeRepository" %in% names(entry)) {
-    if (grepl("ropenscilabs", entry$codeRepository)) {
-      "https://www.repostatus.org/#concept"
-    } else {
-      "https://www.repostatus.org/#active"
-    }
-  } else {
-    ""
+  if (!"codeRepository" %in% names(entry)) {
```

Comment:
simplifying this function that is used for creating a default status (used if there's no repostatus.org badge in the README) :sweat_smile: